src/index.js

```diff
@@ -1,6 +1,6 @@
 import importRule from './rules/import';
 
-export default {
+module.exports = {
```

Comment:
Have you considered using [babel-plugin-add-module-exports](https://www.npmjs.com/package/babel-plugin-add-module-exports) instead?